servatrice/src/servatrice.cpp

```diff
@@ -176,7 +176,8 @@ bool Servatrice::initServer()
     bool registrationEnabled = settingsCache->value("registration/enabled", false).toBool();
     bool requireEmailForRegistration = settingsCache->value("registration/requireemail", true).toBool();
 
-    qDebug() << "Registration enabled: " << regServerOnly;
```

Comment:
On the server startup, some options are printed to the console.
This was reporting a wrong statement mixing up the description of one option with the value of another